bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -2529,14 +2529,16 @@ public int runCmd(CommandLine cmdLine) throws Exception {
                         .isBlank(remoteBookieidToDecommission) ? Bookie.getBookieAddress(bkConf)
                                 : new BookieSocketAddress(remoteBookieidToDecommission));
                 admin.decommissionBookie(bookieAddressToDecommission);
-                LOG.info("The ledgers stored in the given decommissioning bookie are properly replicated");
+                LOG.info("The ledgers stored in the given decommissioning bookie: {} are properly replicated",
+                        bookieAddressToDecommission);
                 runFunctionWithRegistrationManager(bkConf, rm -> {
                     try {
                         Versioned<Cookie> cookie = Cookie.readFromRegistrationManager(rm, bookieAddressToDecommission);
                         cookie.getValue().deleteFromRegistrationManager(rm, bookieAddressToDecommission,
                                 cookie.getVersion());
                     } catch (CookieNotFoundException nne) {
-                        LOG.warn("No cookie to remove, it could be deleted already : ", nne);
+                        LOG.warn("No cookie to remove for the decommissioning bookie: " + bookieAddressToDecommission
```

Comment:
can you change it to use `{}`

Reply:
@sijie considered about it, but in the Logger class there is no warn method which takes Object arg and Throwable t as arguments (I mean void warn(String format, Object arg1, Throwable t)). The only warn method which takes Throwable as argument is this - https://www.slf4j.org/api/org/slf4j/Logger.html#warn(java.lang.String,%20java.lang.Throwable)

Reply:
https://www.slf4j.org/api/src-html/org/slf4j/Logger.html#line.509 doesn't this work for you?

Reply:
changed as we discussed.